### PR TITLE
docs(provider/kubernetes) - Adding additional tool tips for kubernetes v2 provider

### PR DESCRIPTION
**Pull request by imosquera**
Adding additional tool tips for kubernetes v2 provider.

### CODE REVIEW

File: app/scripts/modules/core/src/artifact/expectedArtifactMultiSelector.component.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ class ExpectedArtifactMultiSelectorComponent implements IComponentOptions {
         <ng-form name="artifacts">
           <div class="col-md-12">
             <div class="form-group">
-              <label class="col-md-3 sm-label-right">{{ctrl.artifactLabel}}</label>
+              <label class="col-md-3 sm-label-right">{{ctrl.artifactLabel}}<help-field key="kubernetes.manifest.requredArtifactsToBind"/></label>
```
Comment (lwander) [inline app/scripts/modules/core/src/artifact/expectedArtifactMultiSelector.component.ts]: Can you make these tool tips parameters? This selector is built to be used in other places too (e.g. upcoming rosco & appengine work).

Comment (lwander) [reply app/scripts/modules/core/src/artifact/expectedArtifactMultiSelector.component.ts]: Also, typo in the word "required".

File: app/scripts/modules/core/src/artifact/expectedArtifactSelector.component.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ class ExpectedArtifactSelectorComponent implements IComponentOptions {
       <div class="container-fluid form-horizontal">
         <ng-form name="artifact">
           <div class="form-group">
-            <label class="col-md-3 sm-label-right">Expected Artifact</label>
+            <label class="col-md-3 sm-label-right">Expected Artifact <help-field key="kubernetes.manifest.expectedArtifact"/></label>
```
Comment (lwander) [inline app/scripts/modules/core/src/artifact/expectedArtifactSelector.component.ts]: Same comment here about parameters

Comment (lwander) [reply app/scripts/modules/core/src/artifact/expectedArtifactSelector.component.ts]: Actually, as long as the text isn't k8s specific I would just namespace these tooltips differently.

File: app/scripts/modules/kubernetes/src/help/kubernetes.help.ts [hunk 3]
```diff
@@ -164,6 +164,8 @@ const helpContents: {[key: string]: string} = {
         <b>artifact:</b> The manifest is read from an artifact supplied/created upstream. The expected artifact must be referenced here, and will be bound at runtime.
       </p>
   `,
+  'kubernetes.manifest.expectedArtifact': 'The artifact that is to be applied to the Kubernetes account for this stage.  The artifact should represent a valid Kubernetes manfiest.',
```
Comment (lwander) [inline app/scripts/modules/kubernetes/src/help/kubernetes.help.ts]: Typo in last word (manifest)

File: app/scripts/modules/kubernetes/src/help/kubernetes.help.ts [hunk 4]
```diff
@@ -164,6 +164,8 @@ const helpContents: {[key: string]: string} = {
...
+  'kubernetes.manifest.requredArtifactsToBind': 'These artifacts must be present in the context for this stage to successfully complete. Artifacts specfied will be <a href="https://www.spinnaker.io/reference/artifacts/in-kubernetes-v2/#binding-artifacts-in-manifests">bound to the deployed manfiest.</a>',
```
Comment (lwander) [inline app/scripts/modules/kubernetes/src/help/kubernetes.help.ts]: Typo in last word (manifest)

**Comment by imosquera** [comment]
I made all the suggested changes.  I left the tool tips kubernetes specific so they can be more helpful @lwander

**Pull request merged**